docs/UserGuide.md

```diff
@@ -15,17 +16,21 @@ WedLinker is a **desktop app for managing contacts, optimized for use via a  Lin
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Ensure you have `Java 17` or above installed in your Computer. If you do not have Java 17 installed, or you are unsure, refer to the guide below.
```

Comment:
I remember Dr. Wee asking "How can users know whether they have Java 17?" -- Maybe can add instruction to "Open terminal and run `java -version`, or something similar?